go.mod

```diff
@@ -6,32 +6,120 @@ require (
 	github.com/davecgh/go-spew v1.1.1
 	github.com/fatih/color v1.12.0
 	github.com/ghodss/yaml v1.0.0
-	github.com/go-logr/logr v0.4.0
+	github.com/go-logr/logr v1.2.0
 	github.com/gofrs/uuid v3.3.0+incompatible
 	github.com/google/go-cmp v0.5.6
 	github.com/gorilla/websocket v1.4.2
-	github.com/gosuri/uilive v0.0.4 // indirect
 	github.com/gosuri/uiprogress v0.0.1
 	github.com/gosuri/uitable v0.0.4
 	github.com/hashicorp/go-multierror v1.1.1
-	github.com/manifoldco/promptui v0.8.0
-	// using latest commit from 'github.com/openshift/api@release-4.9'
-	github.com/openshift/api v0.0.0-20211028023115-7224b732cc14
-	github.com/operator-framework/api v0.13.0
+	github.com/manifoldco/promptui v0.9.0
+	github.com/openshift/api v0.0.0-20220912162045-8cfbc78b21bd
+	github.com/operator-framework/api v0.17.1
 	github.com/pkg/errors v0.9.1
-	github.com/prometheus/client_golang v1.12.0
+	github.com/prometheus/client_golang v1.12.1
 	github.com/prometheus/client_model v0.2.0
 	github.com/prometheus/common v0.32.1
 	github.com/redhat-cop/operator-utils v1.3.3-0.20220121120056-862ef22b8cdf
-	github.com/spf13/cobra v1.2.1
+	github.com/spf13/cobra v1.4.0
 	github.com/spf13/viper v1.8.1
-	github.com/stretchr/testify v1.7.0
-	k8s.io/api v0.22.7
-	k8s.io/apimachinery v0.22.7
-	k8s.io/client-go v0.22.7
-	k8s.io/kubectl v0.22.7
-	k8s.io/metrics v0.22.7
-	sigs.k8s.io/controller-runtime v0.10.3
+	github.com/stretchr/testify v1.8.0
+	k8s.io/api v0.24.2
+	k8s.io/apimachinery v0.24.2
+	k8s.io/client-go v0.24.2
+	k8s.io/kubectl v0.24.2
+	k8s.io/metrics v0.24.2
+	sigs.k8s.io/controller-runtime v0.12.1
 )
 
-go 1.16
+require (
```

Comment:
quite a few additional dependencies with the upgrade!? :)

Reply:
yes, `go mod tidy` generated a lot of changes in `go.mod` 🤷‍♂️